Formula/llvm.rb

```diff
@@ -209,11 +266,11 @@ def install
     if build.with? "clang"
       (share/"clang/tools").install Dir["tools/clang/tools/scan-{build,view}"]
       if build.head?
-        inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
+        # inreplace "#{share}/clang/tools/scan-build/bin/scan-build", "$RealBin/bin/clang", "#{bin}/clang"
```

Comment:
It seems this was probably necessary so it could find `clang` after installation into a non-bin directory. If it runs fine: you can remove these.